src/Styles/Login/LoginPage.css

```diff
@@ -21,19 +22,20 @@
 /*Form Div CSS*/
 .loginFormMainDiv {
   background-color: var(--primary-background);
-  width: 28.5%;
-  padding: 1% 1% 1.5% 1%;
-  margin: 4% 2%;
+  width: 405px;
+  height: 330px;
+  padding: 8px 8px 8px 8px;
```

Comment:
padding is 8px for all sides right then you could have written 8px once why 4 times?